vibi-dpu/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "vibi-dpu"
+version = "1.0.0"
+edition = "2021"
+authors = ["Tapish Rathore <tapish@vibinex.com>"]
+license = "GPL-3.0-or-later"
+description = "Vibinex Data Processing Unit for analysing praivate data"
+readme = "README.md"
+homepage = "https://vibinex.com"
+repository = "https://github.com/Alokit-Innovations/vibi-dpu"
+keywords = ["pullrequest", "privacy", "git"]
+categories = ["development-tools", "profiling"]
```

Comment:
Did you get these category names from the crates' website?

Reply:
Yep